### DISCUSSION TITLE
How to create zones of interest with sleap

### DISCUSSION THREAD

**Discussion by natacharossi** [talmolab/sleap#1692] [Help!]
Hi, 

I would like to create zones on my camera views during of after tracking with sleap that correspond to the flowers with which the bees of my experiment interacted. Does anyone know how to do that?

Thanks,

**Comment by Marielamend** [comment]
Hi @frenchnat!

Here is a similar post to your inquiry and what we suggested!
> 
 > To tackle this, you need to train two separate models with different skeletons
> 
(one consisting of your bee skeleton and the other representing the flowers)
> 
 > and then run them in the same video. Make sure that each model with its respective skeleton is made in a separate project. This way you can get the coordinates you want for each instance.
 > #1645
> 
For post-processing, you can check out [DeepOF](https://deepof.readthedocs.io/en/latest/) where you can extract trajectories and distances from your projects!
> 


Hope this helps!

Best, 
Mariela

**Reply by natacharossi** [reply]
Hi Mariela,

Thanks a lot for your answer. I thought about doing this but a skeleton did not really define a zone in my head, should I make the skeleton circular then?

All the best,
Natacha

**Reply by sronilsson** [reply]
@frenchnat I had possibly related problem recently… I solved it, not by playing with sleap parameters, but by post-hoc  e.g., [drawing circle](https://simba-uw-tf-dev.readthedocs.io/en/latest/simba.mixins.html#simba.mixins.geometry_mixin.GeometryMixin.bodyparts_to_circle) of a specified radius with the body-part of interest at center, or fit defined body-parts [into a shape geometry](https://simba-uw-tf-dev.readthedocs.io/en/latest/simba.mixins.html#simba.mixins.geometry_mixin.GeometryMixin.bodyparts_to_polygon) then check if/when/by how much those geometries intersects with other geometries of other animals and objects. Took little tinkering though… didn’t try DeepOF, they may have a off-shelf solution

**Reply by natacharossi** [reply]
Alright, I will try that then. Thanks @sronilsson !
